invenio_accounts/sessions.py

```diff
@@ -133,9 +133,7 @@ def delete_session(sid_s):
     # Remove entries from sessionstore
     _sessionstore.delete(sid_s)
     # Find and remove the corresponding SessionActivity entry
-    if request and "_impersonator_id" not in session:
-        with db.session.begin_nested():
-            db.session.query(SessionActivity).filter_by(sid_s=sid_s).delete()
+    db.session.query(SessionActivity).filter_by(sid_s=sid_s).delete()
```

Comment:
the impact when impersonating would be in worst case that the session gets deleted...about the `request` check I think it was added explicitly by @kpsherva, any memory on why it was needed?

Reply:
this is what I found, I cannot remember why otherwise: https://github.com/inveniosoftware/invenio-accounts/pull/472#discussion_r1461686198

Reply:
Then let's test this on our test instances